zipkin-storage/kafka/src/main/java/zipkin2/storage/kafka/KafkaSpanStore.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2015-2018 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package zipkin2.storage.kafka;
+
+import zipkin2.Call;
+import zipkin2.DependencyLink;
+import zipkin2.Span;
+import zipkin2.storage.QueryRequest;
+import zipkin2.storage.SpanStore;
+
+import java.util.List;
+
+/**
+ * This class is intentionally not implemented at this time.
+ */
+public class KafkaSpanStore implements SpanStore {
+    public KafkaSpanStore(KafkaStorage kafkaStorage) {
+    }
+
+    @Override
+    public Call<List<List<Span>>> getTraces(QueryRequest queryRequest) {
+        return null;
```

Comment:
Right now the behavior is that you logs get filled with NPE. This is one of the checklist item I left. I'm not fully understanding how `Call.create()` helps because in positive changes they seem to return `Call.create(null)`. I was planning to ask the community what the preferred way to fail here would be.